exp/addons/api/v1beta1/clusterresourcesetbinding_types.go

```diff
@@ -100,6 +100,14 @@ func (c *ClusterResourceSetBinding) DeleteBinding(clusterResourceSet *ClusterRes
 			break
 		}
 	}
+	// Remove the ClusterResourceSet from the ClusterResourceSetBinding OwnerReferences.
+	for i, ownerRef := range c.OwnerReferences {
```

Comment:
Previously we weren't updating ownerReferences at all after creation so the initial ClusterResourceSet was always an owner even if it had been deleted.